analytics/docker-compose.yml

```diff
@@ -0,0 +1,49 @@
+version: "3.7"
+
+services:
+  # Matomo doesn't work with psql, so we use mysql instead
+  # See https://matomo.org/faq/how-to-install/faq_55/
```

Comment:
@microwavenby unfortunately, it looks like the Postgres adoption is [never going to happen](https://github.com/matomo-org/matomo/issues/500) because its been requested since 2009😢